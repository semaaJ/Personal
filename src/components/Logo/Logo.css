.logo {
    display: flex;
    flex-direction: column;
    align-items: center;
}

.logoImage {
    margin: 50px auto 15px auto;
}

.name {
    background-color: #383737;
    color: white;
    text-transform: uppercase;
    font-size: 18px;
    font-weight: bold;
    width: 170px;
    padding-top: 3px;
    padding-bottom: 3px;
    text-align: center;
}