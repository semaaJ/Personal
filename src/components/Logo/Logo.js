import React from 'react';
import face from './face.png';
import './Logo.css';

const Logo = () => {
    return (
        <div className="logo">
            <img className="logoImage" src={face} />
            <div className="name">JAMES MILES</div>
        </div>
    )
}   

export default Logo;