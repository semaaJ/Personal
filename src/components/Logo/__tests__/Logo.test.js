import React from 'react';
import Logo from '../Logo';
import { shallow } from 'enzyme';
import '../../../__mocks__/intersectionObserverMock';
import "../../../setupTests"

test('should render Logo correctly', () => {
    const component = shallow(<Logo />);
    expect(component.html()).toMatchSnapshot();
});
