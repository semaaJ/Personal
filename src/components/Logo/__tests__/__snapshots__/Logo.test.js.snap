// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Logo correctly 1`] = `"<div class=\\"logo\\"><img class=\\"logoImage\\" src=\\"face.png\\"/><div class=\\"name\\">JAMES MILES</div></div>"`;
